components/01-atoms/lists/_list.scss

```diff
@@ -3,3 +3,33 @@
   margin: 0;
   padding: 0;
 }
+
+ul {
+  list-style-type: disc;
+
+  ul {
+    list-style-type: circle;
+
+    ul {
+      list-style-type: square;
+    }
+  }
+}
+
+ol {
+  list-style-type: decimal;
+
+  ol {
+    list-style-type: lower-alpha;
+
+    ol {
+      list-style-type: lower-roman;
+    }
+  }
+}
+
+.text-field {
```

Comment:
These styles won't be applied in Drupal as `.text-field` isn't a class that's added by default to formatted text fields. We can discuss via zoom. I don't think this should block merging.

Reply:
The `text-field.twig` molecule adds this class. I intend to use that on anything that uses a wysiwyg.